### ISSUE TITLE
Don't pass arguments to ArgumentParser since it fails with Python 3.5

### ISSUE THREAD

**Issue by gfa** [Roguelazer/muttdown#12]
Hi

Calling ArgumentParser with version parameter doesn't work when running under python 3.5, so I patched it out.

I don't see any regression when applying the patch and running with Python 2.7

```
--- a/muttdown/main.py
+++ b/muttdown/main.py
@@ -122,7 +122,7 @@ def smtp_connection(c):
 
 
 def main():
-    parser = argparse.ArgumentParser(version='%s %s' % (__name__, __version__))
+    parser = argparse.ArgumentParser()
     parser.add_argument(                                                                                                                                                                     
         '-c', '--config_file', default=os.path.expanduser('~/.muttdown.yaml'),                                                                                                               
         type=argparse.FileType('r'), required=False,                        
```

**Comment by Roguelazer** [comment]
Blargh of course argparse isn't consistent between versions.

I'll probably conditionalize on `sys.version_info` so we can keep the nice argparse version message on python 3.6.

**Comment by Roguelazer** [comment]
Fixed in version 0.3.1
